### ISSUE TITLE
Romanian translation

### ISSUE THREAD

**Issue by ainzz** [squidfunk/mkdocs-material#1438]
## Instructions

1. Check, if your language is already available: [here](http://bit.ly/2DCzaL0)
2. If it isn't, please translate the labels on the right:

``` jinja
{% macro t(key) %}{{ {
  "language": "ro",
  "direction": "ltr",
  "clipboard.copy": "Copiază în clipboard",
  "clipboard.copied": "Copiat în clipboard",
  "edit.link.title": "Editeaza această pagină",
  "footer.previous": "Anterior",
  "footer.next": "Următor",
  "meta.comments": "Comentarii",
  "meta.source": "Sursă",
  "search.language": "ro",
  "search.pipeline.stopwords": true,
  "search.pipeline.trimmer": true,
  "search.placeholder": "Căutare",
  "search.result.placeholder": "Tastează pentru a începe căutarea",
  "search.result.none": "Nu a fost găsit niciun document",
  "search.result.one": "1 document găsit",
  "search.result.other": "# documente găsite",
  "search.tokenizer": "[\s\-]+",
  "skip.link.title": "Sari la conținut",
  "source.link.title": "Accesează repository-ul",
  "source.revision.date": "Ultima actualizare",
  "toc.title": "Cuprins"
}[key] }}{% endmacro %}
```

Thanks!

**Comment by squidfunk** [comment]
Added in 0730aae9.

**Comment by squidfunk** [comment]
Released as part of 4.6.2.
